runtime/src/accounts_db.rs

```diff
@@ -5433,33 +5433,21 @@ impl AccountsDb {
 
     fn find_storage_candidate(&self, slot: Slot, size: usize) -> Arc<AccountStorageEntry> {
```

Comment:
I think we will likely get rid of this behavior completely where we can find an append vec and add to it or add a new one.